SU-HSDO/suhumsci
# shs-131: ensure l2 no links dont behave like links

## Summary
This adjusts level 2 mega menu items so that items are not links have no hover styles and look like other L2s

## Steps to Test
1. On a site with the new megamenu enabled, edit a level two menu item that has children in Drupal. Since a link is required in the link field, add `<nolink>` save.
2. View the new megamenu and ensure that the L2 no link item: doesn't have a hover state and has the same `font-weight` as the other L2s

## Screenshots
![Screen Shot 2022-12-13 at 2 32 35 PM](https://user-images.githubusercontent.com/8405274/207428543-827c6265-ac0a-4b34-ab72-ea563dfd0761.png)
